Automattic/simplenote-ios
# Resigning SearchBar's Responder status when opening Notes

### Fix
In this PR we're resigning the SearchBar's First Responder status, as a workaround, in order to prevent the "Transparent Keyboard" glitch.

Sample video of the actual bug attached below.

cc @bummytime WDYT?

Closes #616

### Test
1. Laungh the app
2. Press over the Search Bar to enter Search Mode
3. Press over any of your notes
4. Verify the keyboard gets dismissed as soon as you press over any row
5. Swipe back to dismiss the editor
6. Verify the keyboard does not show up again on its own
7. Verify that pressing over the SearchBar shows the Keyboard, without further issues

### Release
No release notes required!

---

### Bug's Demo

![Bug](https://user-images.githubusercontent.com/1195260/73768676-3039a600-4758-11ea-8b20-2bdd96e4e8c6.gif)
